lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -682,6 +688,10 @@ def add_mastercard_fields(xml, credit_card, parameters, three_d_secure)
         add_mc_ucafind(xml, credit_card, three_d_secure)
       end
 
+      def add_third_party_token(xml, parameters)
```

Comment:
Same here, looks like is not used in the method the parameter `parameters`.

Reply:
yes I will update it, parameters is not needed, thanks

Reply:
Same suggestion here, related to defining a method for a single parameter.